README.md

```diff
@@ -1,3 +1,28 @@
-# tpm_module
+# TPM2 module schematics
 
-TPM module PCB project for Libretrend LT1000
\ No newline at end of file
+TPM2 module PCB project for platforms with 2x5 pin, 2mm pitch LPC header, e.g.
+Libretrend LT1000.
+
+## Description
+
+Trusted Platform Module2 compatible with 2x5p LPC header (populated on Librebox
+platform) can be used in:
+
+* disk encryption
+* password protection
+* platform integrity and other security issues
+
+TPM2 module for Librebox features:
+
+* TPM SLB9665TT20FW561XUMA1
+* Operating voltage 3-3.3V
+* Meeting Intel TXT, Microsoft Windows and Google Chromebook certification
+  criteria for successful platform qualification
+* True Random Number Generator (TRNG)
+* Full personalization with Endorsement Key (EK) and EK certificate
+* Supports the LPC interface
+
+## Where I can buy this product?
+
+If you are not the enthusiast of DYI, then the plug and play product can be
+bought in the [3mdeb shop](https://shop.3mdeb.com/product/tpm2-module-for-librebox/).
```

Comment:
@artur-rs question if Libretrend selling this module - we should add link.

Reply:
added